tests/ui/test_ui_tools.py

```diff
@@ -2079,16 +2079,16 @@ def test_footlinks_view(self, message_fixture, message_links,
             assert footlinks is None
             assert not hasattr(footlinks, 'original_widget')
 
-    @pytest.mark.parametrize('footlinks_enabled, expected_instance', [
-        (False, type(None)),
-        (True, Padding),
+    @pytest.mark.parametrize('maximum_footlinks, expected_instance', [
+        (0, type(None)),
+        (3, Padding),
```

Comment:
I think it would be reasonable to add another case here, which doesn't correspond to the enabled/disabled pair.